dataactvalidator/scripts/loadFile.py

```diff
@@ -177,6 +177,8 @@ def load_program_activity(filename):
             {"program_activity_code": {"pad_to_length": 4}, "agency_id": {"pad_to_length": 3},
              "allocation_transfer_id": {"pad_to_length": 3, "keep_null": True}, "account_number": {"pad_to_length": 4}}
         )
+        # Lowercase Program Activity Name
+        data['program_activity_name'] = data['program_activity_name'].apply(lambda x: x.lower())
```

Comment:
Just curious, does the `apply` not update `data['program_activity_name']` in place? It returns a new object?